rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.adoc

```diff
@@ -13,7 +13,8 @@ include::modules/rosa-policy-shared-responsibility.adoc[leveloffset=+1]
 
 [id="additional-resources-rosa-policy-responsibility-matrix"]
```

Comment:
I don't think it causes problems if you leave this here, but since you switched to a trailing include this can be cut.